stripe/resource.py

```diff
@@ -312,9 +312,32 @@ def instance_url(self):
 
 class ListObject(StripeObject):
 
-    def all(self, **params):
+    def list(self, **params):
```

Comment:
Hah, almost a little comical how closely this naming schema matches Ruby :) Not a bad thing though!